web/src/components/snapshots/SnapshotStorageDestination.tsx

```diff
@@ -32,13 +32,13 @@ type CACertificate = {
 };
 type State = {
   azureBucket?: string;
-  azureClientId?: string;
-  azureClientSecret?: string;
+  azureClientId: string;
+  azureClientSecret: string;
   azurePath?: string;
-  azureResourceGroupName?: string;
-  azureStorageAccountId?: string;
-  azureSubscriptionId?: string;
-  azureTenantId?: string;
```

Comment:
Wanted to remove the `undefined` option from most of these since the app won't work if they are undefined.